cde-pentaho5/resource/resources/custom/amd-components/Dashboard/component.xml

```diff
@@ -64,6 +65,19 @@
             <Version>1.0</Version>
           </Header>
         </DesignerProperty>
+        <DesignerProperty>
+          <Header>
+            <Name>oneWayMap</Name>
+            <Parent>BaseProperty</Parent>
+            <DefaultValue>"false"</DefaultValue>
+            <Description>One Way Map</Description>
+            <Tooltip>If 'true' parameter propagation will only go in one direction</Tooltip>
```

Comment:
Can we make it apparent which is the direction? Something like:
`If 'true' parameter propagation will only go from this Dashboard, to the required one`
@afrjorge comment on the above please